org.gazebosim.Gazebo.yaml

```diff
@@ -0,0 +1,720 @@
+  - name: SDFormat9
+    buildsystem: cmake
+    config-opts:
+      - -DLIB_INSTALL_DIR:PATH=/app/lib
+      - -DBUILD_TESTING:BOOL=OFF
+    build-options:
+      env:
+        LIBRARY_PATH: "/app/lib:/usr/lib"
+        LD_LIBRARY_PATH: "/app/lib:/usr/lib"
+    sources:
+      - type: archive
+        url: https://github.com/osrf/sdformat/archive/refs/tags/sdformat9_9.5.0.tar.gz
+        sha256: 53791427b25c2485a2f74aab90e07ff8e7470f86532ecd30b9510bbec4122c97
+        
+    # need ld
+    # Enable PROFILER to work around freezing at start.
+    # https://github.com/osrf/gazebo/issues/2457
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1871291
+  - name: Gazebo
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE:STRING=Release
+      - -DCMAKE_INSTALL_PREFIX:PATH=/app
+      - -DBUILD_TESTING:BOOL=ON
+      - -DENABLE_PROFILER:BOOL=ON
+    build-options:
+      env:
+        LIBRARY_PATH: "/app/lib:/usr/lib"
+        LD_LIBRARY_PATH: "/app/lib:/usr/lib"
+    #post-install:
+      #- install -Dm644 gazebo/gui/images/gazebo.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      #- install -Dm644 gazebo128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      #- install -Dm644 gazebo/gui/images/gazebo.svg ${FLATPAK_DEST}/share/app-info/icons/flatpak/scalable/${FLATPAK_ID}.svg
+      #- install -Dm644 gazebo128x128.png ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128/${FLATPAK_ID}.png
+      #- install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      #- install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      #- export IGN_IP=127.0.0.1
+      #- export GAZEBO_IP=127.0.0.1
+      #- ctest --verbose --parallel 1 --exclude-regex ".*OpenAL.*" || exit 0
+    sources:
+      - type: archive
+        url: https://github.com/osrf/gazebo/archive/refs/tags/gazebo11_11.5.0.tar.gz
+        sha256: cd46d85e01aa40ad181a7b0afcbe23dfd13ecdfb6ea37b881655205d8fc1aec0
+      #- type: file
+        #path: org.gazebosim.Gazebo.metainfo.xml
+        # Protobuf patch adopt from https://github.com/osrf/gazebo/issues/2639
+      - type: patch
+        path: Gazebo_fix_protobuf_linking.patch
+        # Create png icon with inkscape, the original icon is not square, must resize unproportionally.
+      #- type: file
+        #path: gazebo128x128.png
+      #- type: file
+        #path: org.gazebosim.Gazebo.desktop
+      #- type: patch
+        #path: fix_Ogre_ShadowExtrudePointLight_not_found.patch
+        
+  - name: install-desktop-icon
+    buildsystem: simple
+    build-commands:
+      - ls -l
+      #- cp -r /app/share/gazebo-11/media /app/share/gazebo
+      #- cp -r /app/share/gazebo-11/models /app/share/gazebo
+      #- cp -r /app/share/gazebo-11/worlds /app/share/gazebo
+      - install -Dm644 gazebo/gui/images/gazebo.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
```

Comment:
there is no such file as this is a different module. Hence the failure.
You will have only two file. `org.gazebosim.Gazebo.metainfo.xml` and `setup.sh`. They are listed below.